Tests/libhostmgrTests/resources/dotenv-fixtures.env

```diff
@@ -0,0 +1,26 @@
+UNQUOTED_STRING=foo
+QUOTED_STRING="foo"
+
+UNQUOTED_STRING_WITH_LEADING_SPACE_IN_VALUE= foo
+UNQUOTED_STRING_WITH_TRAILING_SPACE_IN_VALUE=foo 
+
+STRING_WITH_QUOTED_LEADING_SPACE=" foo"
+STRING_WITH_QUOTED_TRAILING_SPACE="foo "
+
+STRING_WITH_LEADING_QUOTE=""foo"
+UNQUOTED_STRING_WITH_EMBEDDED_QUOTE=foo"bar
+STRING_WITH_TRAILING_QUOTE="foo""
+STRING_WITH_EMOJI="🛠 Build for Testing"
+
+STRING_WITH_NO_VALUE=
+STRING_WITH_WHITESPACE_VALUE=
+
+UNQUOTED_STRING_WITH_EQUAL_SIGN=foo=bar
+QUOTED_STRING_WITH_EQUAL_SIGN="foo=bar"
+
+IGNORED_STRING_WITH_NO_EQUAL_SIGN
+
+COMMENTED_QUOTED_STRING="foo" # This is a comment
+COMMENTED_UNQUOTED_STRING=foo # This is a comment
+
+QUOTED_STRING_WITH_NEWLINE="foo\nbar"
```

Comment:
I feel like it might be good to cover more cases (multi-line string, mixed single and double quotes, etc) in the fixture file (we could get some inspiration from other implementations like [vlucas/phpdotenv](https://github.com/vlucas/phpdotenv/blob/v5.5.0/tests/fixtures/env/multiline.env) and [motdotla/dotenv](https://github.com/motdotla/dotenv/blob/v16.0.3/tests/.env-multiline)?), what do you think?

Reply:
+ 💯 , especially since some of our mobile-secrets _do_ use multiline values in some cases